Sources/PackageLoading/Target+PkgConfig.swift

```diff
@@ -148,6 +150,13 @@ extension SystemPackageProviderDescription {
             if case .linux(.fedora) = platform {
                 return true
             }
+        case .nuget:
+            switch platform {
+            case .darwin, .windows, .linux:
```

Comment:
should this be windows only? is there formal support and practical adoption of nuget on linux and darwin?

Reply:
Yes, they are formally supported and adopted - even though the primary usage is .NET, but that is more a historical artifact.

Reply:
A nice example package: https://www.nuget.org/packages/runtime.debian.8-x64.runtime.native.System.Security.Cryptography.OpenSsl/

Prebuilt .so for Debian x86_64.